gui/autodump.lua

```diff
@@ -306,14 +306,50 @@ function Autodump:onRenderFrame(dc, rect)
     end
 end
 
+local function tile_props(pos, tt) --Returns is_ground, is_open_air
+    local shape_attrs = df.tiletype_shape.attrs[df.tiletype.attrs[tt].shape]
+    if shape_attrs.walkable then --TODO: don't dump on statues, etc.?
```

Comment:
dumping on statues is probably fine. the game allows it, so we can too.

Reply:
Should we allow dumping on fortifications? Projectiles can end up there.

Reply:
I'd say allow it. If the player selects a spot that is even remotely viable, then we should respect it